apps/studio/src/features/mail/templates/templates.ts

```diff
@@ -50,11 +54,47 @@ export const invitationTemplate = (
   }
 }
 
+export const publishAlertContentPublisherTemplate = (
+  data: PublishAlertContentPublisherEmailTemplateData,
+): EmailTemplate => {
+  const { recipientEmail, siteName, resource } = data
+  const resourceStudioUrl = getResourceStudioUrl(resource)
+
+  return {
+    subject: `[Isomer Studio] ${resource.title} has been published`,
+    body: `<p>Hi ${recipientEmail},</p>
+<p>You have successfully published "${resource.title}" on ${siteName}. You can access your published content on Isomer Studio at <a href="${resourceStudioUrl}">${resourceStudioUrl}</a>.</p>
+<p><strong>Note:</strong> You're receiving this notification because content was published during a Singpass authentication outage. If you didn't authorize this publication, please contact <a href="${ISOMER_SUPPORT_LINK}">${ISOMER_SUPPORT_EMAIL}</a> immediately.</p>
+<p>Best,</p>
+<p>Isomer team</p>`,
+  }
+}
+
+export const publishAlertSiteAdminTemplate = (
+  data: PublishAlertSiteAdminEmailTemplateData,
+): EmailTemplate => {
+  const { recipientEmail, publisherEmail, siteName, resource } = data
+  const resourceStudioUrl = getResourceStudioUrl(resource)
```

Comment:
[nit] `studioReosurceUrl`

Reply:
updated in https://github.com/opengovsg/isomer/pull/1349/commits/9faba3d4858aff37c312dbc18501fc9bf8821e57 !